com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs

```diff
@@ -566,6 +566,39 @@ public override void Setup(ScriptableRenderContext context, ref RenderingData re
                 ConfigureCameraTarget(activeColorRenderTargetId, activeDepthRenderTargetId);
             }
 
+            bool renderingToBackBufferTarget = camera.targetTexture == null;
+
+            // If Camera Renders to backbuffer and MSAA Sample Count is larger than 1 we may have to disable/enable MSAA backbuffer
+            // The condition depends on whether the intermediate MSAA render targets has been created or not
+            if (cameraTargetDescriptor.msaaSamples > 1 && renderingToBackBufferTarget)
+            {
+                int qualitySettingsMsaaSampleCount = QualitySettings.antiAliasing > 0 ? QualitySettings.antiAliasing : 1;
+
+                if (createColorTexture)
+                {
+                    if (qualitySettingsMsaaSampleCount > 1)
+                    {
+                        QualitySettings.antiAliasing = 1;
+#if ENABLE_VR && ENABLE_XR_MODULE
+                        XRSystem.UpdateMSAALevel(1);
```

Comment:
I think we should not update XR MSAA level here as that could potentially trigger eye texture reallocation. Also in some cases MSAA eye texture is desired for final XR composition pass.